apps/rundown/src/launcher.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import moduleApi = require('module');
+import * as process from 'process';
+
+import { /* type */ LauncherAction, IIpcTrace, IIpcDone, IIpcTraceRecord } from './LauncherTypes';
+
+// The _ipcTraceRecordsBatch will get transmitted when this many items are accumulated
+const IPC_BATCH_SIZE: number = 1000;
```

Comment:
This seems pretty big.

Reply:
Reduced to 300.  If each path string is 100 bytes, then 300*100 = 30k.